ci-operator/config/openshift/installer/openshift-installer-release-4.14__okd.yaml

```diff
@@ -201,30 +201,6 @@ tests:
     - ref: ipi-conf-aws-fcos-ami
     - chain: ipi-install
     workflow: openshift-e2e-aws
-- as: e2e-agent-compact-ipv4
```

Comment:
These jobs were meant to support backporting on 4.14

Reply:
I can't think of any reason why we would backport that.

Reply:
4.14 backport is no more relevant, as ABI OKD was available in 4.15